Source/SwiftLintFramework/Extensions/String+SwiftLint.swift

```diff
@@ -69,4 +69,16 @@ extension String {
         }
         return nil
     }
+
+    var stringByDeletingLastPathComponent: String {
+        return (self as NSString).stringByDeletingLastPathComponent
+    }
+
+    func stringByAppendingPathComponent(pathComponent: String) -> String {
+        return (self as NSString).stringByAppendingPathComponent(pathComponent)
+    }
+
+    public func absolutePathStandardized() -> String {
```

Comment:
Certainly possible. I didn't do much testing other than realizing `stringByStandardizingPath` was necessary and following the pattern from [here](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Extensions/NSFileManager%2BSwiftLint.swift#L18).